config/jobs/kubernetes/sig-k8s-infra/trusted/sig-k8s-infra-test-infra.yaml

```diff
@@ -144,8 +144,11 @@ periodics:
     testgrid-alert-email: k8s-infra-oncall@google.com
     testgrid-num-failures-to-alert: '3'
 - name: ci-test-infra-autobump-prowjobs-for-auto-deploy
-  # This 1h after ci-k8sio-autobump-prow-build-clusters, 2h before ci-test-infra-autobump-prow-for-auto-deploy
-  cron: "30 15-20/5 * * 1-5"  # Bump with label `skip-review`. Run at 08:30 and 13:30 PDT (in UTC) Mon-Fri
+  # prowjob images image is unsafe to automerge
```

Comment:
there's a job above called `ci-test-infra-autobump-prowjobs` that covers just filing PRs that won't auto-merge

Reply:
we validate these now, oh well!

will just delete the job and we can revert the commit later https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/28526/pull-test-infra-prow-checkconfig/1618312335938555904